### PR TITLE
Improve battle overview and gamelog

### PR DESCRIPTION
**Pull request by gereon77**
Relates to #260 

Action panel:
battle1

GameLog:
image

### CODE REVIEW

**Comment by Longwelwind** [comment]
As discussed on Discord, we should keep the `HouseCardChosen` log since if a House Card ability happens after they have been chosen, but before the combat is resolved, it would be better to have a log to specify that they have been chosen.

In general, I think it's best to keep one game log for each action that users have done. We should try to prevent multiple game logs from appearing at once (except when it makes sense or if it's too hard to properly implement a unified log).

File: agot-bg-game-server/src/client/CombatInfoComponent.tsx [hunk 1]
```diff
@@ -38,6 +39,11 @@ export default class CombatInfoComponent extends Component<CombatInfoComponentPr
     render(): ReactNode {
         return (
             <>
+                {this.props.showTitle && (
+                    <div style={{display: 'flex', justifyContent: 'center'}}>
+                        <h5>Battle for <strong>{this.defender.region.name}</strong></h5>
+                    </div>
+                )}
```
Comment (Longwelwind) [inline agot-bg-game-server/src/client/CombatInfoComponent.tsx]: If the title is only shown in the GameState Planel and not in the game log, why not adding it in the `CombatComponent`?

Comment (gereon77) [reply agot-bg-game-server/src/client/CombatInfoComponent.tsx]: Indeed. That would simplify it completely. I close this PR as the branch name does not fit anymore. Creating a new one just for the 3 lines seems a bit ridiculous to me so if you want you could add the title to the combat component by just copying the 3 lines of code from here when you do the highlighting thing?

**Comment by gereon77** [comment]
Closed due to the remarks.

Here are the only valuable lines which could be added with another PR:

```
<div style={{display: 'flex', justifyContent: 'center'}}>
    <h5>Battle for <strong>{this.defender.region.name}</strong></h5>
</div>
```

**Pull request closed without merging**